src/style.css

```
@@ -0,0 +1,58 @@
+html {
+  margin: 0;
+}
+
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  margin-top: 0;
+  margin-bottom: 0;
+  padding: 0;
+}
+
+.stars__star {
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  border-radius: 0.5px;
+  margin-right: 4px;
+  list-style: none;
+  display: flex;
+}
+
+.stars__star:not(:first-child) {
+  background-image: url(./images/star.svg);
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--0 .stars__star {
+  background-image: url(./images/star.svg);
+}
+
+.stars--1 .stars__star:first-child,
+.stars--2 .stars__star:first-child {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 .stars__star:nth-of-type(2) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--3 .stars__star:nth-last-of-type(n+3) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--4 .stars__star:nth-of-type(-n+4) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--5 .stars__star:nth-of-type(-n+5) {
+  background-image: url(./images/star-active.svg);
+}
```

```suggestion
.stars--1 .stars_star:nth-child(-n+1),
.stars--2 .stars_star:nth-child(-n+2),
.stars--3 .stars_star:nth-child(-n+3),
.stars--4 .stars_star:nth-child(-n+4),
.stars--5 .stars_star:nth-child(-n+5) {
  background-image: url(./images/star-active.svg);
}
```
We can also combine it together and use `nth-child` here